styles/main.css

```diff
@@ -0,0 +1,212 @@
+body {
+  background-color: #e5e5e5;
+  align-items: center;
+  font-family: Poppins, sans-serif;
+}
+
+ul {
+  padding-left: 1px;
+  list-style: none;
+}
+
+button:hover {
+  background-color: #a9cbe2;
+  cursor: pointer;
+}
```

Comment:
- [ ] For the interactions there is also a layout that you should follow. Kindly follow this design when hovering these buttons.
### Interaction template in Figma
![image](https://user-images.githubusercontent.com/65068771/149346513-4be59484-ac8f-4e38-8a13-a0bb0622bf1b.png)


Your hover interaction | The template hover interaction 
----------------------|----------------------------
![image](https://user-images.githubusercontent.com/65068771/149346718-555e985e-131d-487c-9502-7c0e8dadb3e0.png) | ![image](https://user-images.githubusercontent.com/65068771/149346818-0e280aa4-c55a-4de0-a402-ca2cff3fc6c7.png)